config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
   devise_for :users
   resources :cars do
+  resources :likes, only: [:create, :destroy]
```

Comment:
Suggestion: On behalf of create and destroy method name you have to rename like to dislike.